src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -487,8 +488,19 @@ export class DataResourceDataProvider implements IGridDataProvider {
 		// interface than the query execution service's saveResults handlers. Here, we take the
 		// format-specific request params (eg, includeHeaders for CSV) and merge the format-agnostic
 		// request params for the serialization request (eg, saveFormat, filePath).
+		let providerId: string;
+		switch (this.cellModel.notebookModel.currentKernelAlias) {
+			case 'Kusto':
+			case 'LogAnalytics':
```

Comment:
You should be using common constants for these - I see a number of places where the provide IDs are hardcoded in so I'd suggest at least combining those to all use the same values to more easily be able to track down the different known aliases and providers. (having separate constants for provider IDs and aliases seems like a good idea giving the casing difference).